src/RNCamera.js

```diff
@@ -11,11 +11,61 @@ import {
   ActivityIndicator,
   Text,
   StyleSheet,
+  PermissionsAndroid,
 } from 'react-native';
 
 import type { FaceFeature } from './FaceDetector';
 
-import { requestPermissions } from './handlePermissions';
+const requestPermissions = async (
+  captureAudio: boolean,
+  CameraManager: any,
+  permissionDialogTitle?: string,
+  permissionDialogMessage?: string,
+): Promise<{ hasCameraPermissions: boolean, hasRecordAudioPermissions: boolean }> => {
+  let hasCameraPermissions = false;
+  let hasRecordAudioPermissions = false;
+
+  let params = undefined;
+  if (permissionDialogTitle || permissionDialogMessage) {
+    params = { title: permissionDialogTitle, message: permissionDialogMessage };
+  }
+
+  if (Platform.OS === 'ios') {
+    hasCameraPermissions = await CameraManager.checkVideoAuthorizationStatus();
+  } else if (Platform.OS === 'android') {
+    const cameraPermissionResult = await PermissionsAndroid.request(
+      PermissionsAndroid.PERMISSIONS.CAMERA,
+      params,
+    );
+    hasCameraPermissions = cameraPermissionResult === PermissionsAndroid.RESULTS.GRANTED;
```

Comment:
Hello! Why you use different "requestPermissions" method for Camera.js and RNCamera.js? Can you add `|| cameraPermissionResult === true` like in Camera.js? Because react-native return boolean-type result for devices before Android 6.0

thx!